units/systemd-journal-upload.service.in

```diff
@@ -26,6 +28,8 @@ ProtectHostname=yes
 ProtectKernelLogs=yes
 ProtectKernelModules=yes
 ProtectKernelTunables=yes
+Restart=on-failure
+RestartSec=15
```

Comment:
Why do you set such long interval?
How about to use `RestartSteps=` and `RestartMaxDelaySec=` ?

Reply:
Ok, I switched this to use `RestartSteps` and `RestartMaxDelaySec`. If there's exponential backoff starting from 100ms, then step 10 should be 51200ms:
```
RestartSteps=10
RestartMaxDelaySec=60 
```